bitnami/solr/values.yaml

```diff
@@ -805,12 +818,6 @@ metrics:
   ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
   ##
   topologySpreadConstraints: {}
-  ## @param metrics.podManagementPolicy Management Policy for Solr Prometheus exporter deployment
```

Comment:
Just beware that this parameter is being removed, in case it was done by an automation.

Reply:
Thanks for the heads up Marcos! This parameter is being removed because it wasn't being used anywhere.

We use a Deployment for metrics, while `podManagementPolicy` is only used in Statefulsets